README-Prototype.md

```diff
@@ -0,0 +1,195 @@
+# esdl Tool
+
+The esdl tool aids with creating and managing ESDL-based and Dynamic ESDL services on an HPCC cluster. It consists of several different commands.
+
+## Overall notes for Tim and Tony
+
+Here is my overview of the design direction:
+
+1. Think of the manifest as config-as-code. The operator using the manifest to generate output shouldn't need to know which options to pass on the command line.
+2. Collapse the template file into the manifest.
+3. Think of the template as having two kinds of markup
```

Comment:
Picky, but after saying "collapse the template into the manifest" let's not refer to it as template anymore, it's the manifest.  My only real point being that templates sound like they are for more generic purposes than these manifests are.  These manifests describe actual service deployments and are only modified by providing variables/values.

Reply:
Ok yep I missed those references.